xmind/tests/test_workbook_mixin_element.py

```diff
@@ -61,140 +58,135 @@ def test_excessive_parameters(self):
+
         obj = WorkbookMixinElement("test", self._ownerWorkbook)
 
         self.assertEqual(obj.getModifiedTime(), "time")
-        self._getAttribute_method.assert_called_once_with('timestamp')
+        self._getAttribute_method.assert_called_once_with(obj, ATTR_TIMESTAMP)
         self._readable_time_method.assert_called_once_with(1)
+        self._assert_init_with_object(obj, "test")
 
     def test_set_modified_time(self):
         """Test setModifiedTime method, input parameter is number"""
-        self._registOwnerWorkbook_method = self._init_patch_with_name(
-            '_registOwnerWorkbook', 'xmind.core.mixin.WorkbookMixinElement.registOwnerWorkbook')
         self._setAttribute__method = self._init_patch_with_name(
-            '_setAttribute', 'xmind.core.Element.setAttribute')
+            '_setAttribute', 'xmind.core.Element.setAttribute', autospec=True)
         obj = WorkbookMixinElement("test", self._ownerWorkbook)
 
         obj.setModifiedTime(1234)
-        self._setAttribute__method.assert_called_once_with("timestamp", 1234)
+        self._setAttribute__method.assert_called_once_with(
+            obj, "timestamp", 1234)
+        self._assert_init_with_object(obj, "test")
 
     def test_set_modified_time_throws(self):
         """Test setModifiedTime method, input parameter is NOT number"""
-        self._registOwnerWorkbook_method = self._init_patch_with_name(
-            '_registOwnerWorkbook', 'xmind.core.mixin.WorkbookMixinElement.registOwnerWorkbook')
         self._setAttribute__method = self._init_patch_with_name(
-            '_setAttribute', 'xmind.core.Element.setAttribute', Exception("ValueError"))
+            '_setAttribute', 'xmind.core.Element.setAttribute',
+            thrown_exception=Exception("super error"), autospec=True)
         obj = WorkbookMixinElement("test", self._ownerWorkbook)
 
         with self.assertRaises(Exception) as ex:
-            obj.setModifiedTime(None)
+            obj.setModifiedTime(0)
 
-        self._setAttribute__method.assert_not_called()
+        self.assertTrue(ex.exception.args[0].find(
+            "super error") == 0)
 
-        self.getLogger().warning("Exception: %s", ex.exception)
+        self._setAttribute__method.assert_called_once_with(
+            obj, ATTR_TIMESTAMP, 0)
+        self._assert_init_with_object(obj, "test")
 
     def test_update_modified_time(self):
         """Test updateModifiedTime method"""
-        self._registOwnerWorkbook_method = self._init_patch_with_name(
-            '_registOwnerWorkbook', 'xmind.core.mixin.WorkbookMixinElement.registOwnerWorkbook')
         self._get_current_time_method = self._init_patch_with_name(
-            '_get_current_time', 'xmind.utils.get_current_time', return_value=12345)
+            '_get_current_time', 'xmind.utils.get_current_time', return_value=12345, autospec=True)
         self._setAttribute__method = self._init_patch_with_name(
```

Comment:
I think we should patch the setModifiedTime method for concentrating on 1 tested method of class.